windows/devices/uwb/UwbSession.cxx

```diff
@@ -14,12 +16,152 @@ UwbSession::UwbSession(std::weak_ptr<uwb::UwbSessionEventCallbacks> callbacks, w
     m_handleDriver(std::move(handleDriver))
 {}
 
+template <class T>
+std::unique_ptr<UWB_APP_CONFIG_PARAM>
+GetUWB_APP_CONFIG_PARAM(const T &value)
+{
+    auto pdata = std::make_unique<uint8_t>(FIELD_OFFSET(UWB_APP_CONFIG_PARAM, paramValue[sizeof T]));
+}
+
+template <class T>
+UWB_SET_APP_CONFIG_PARAMS
+GetUWB_SET_APP_CONFIG_PARAMS()
+{
+}
+
+/**
+ * @brief This function only works for integer types, aka uint8_t, uint16_t, uint32_t, uint64_t
+ *
+ * @param value
+ * @param size
+ * @param dst
+ * @return uint8_t*
+ */
+uint8_t *
+WriteBytes(uint64_t value, std::size_t size, uint8_t *dst)
```

Comment:
Could this not be replaced with `std::memcpy` and then there would be no restrictions on the input?